vm/isolated/isolated.go

```diff
@@ -172,29 +190,39 @@ func (inst *instance) ssh(command string) error {
 }
 
 func (inst *instance) repair() error {
-	log.Logf(2, "isolated: trying to ssh")
-	if err := inst.waitForSSH(30 * time.Minute); err == nil {
-		if inst.cfg.TargetReboot {
-			log.Logf(2, "isolated: trying to reboot")
-			inst.ssh("reboot") // reboot will return an error, ignore it
-			if err := inst.waitForReboot(5 * 60); err != nil {
-				log.Logf(2, "isolated: machine did not reboot")
+	if inst.cfg.TargetReboot {
+		log.Logf(2, "isolated: trying to ssh")
+		if err := inst.waitForSSH(30 * time.Minute); err == nil {
+			log.Logf(2, "isolated: ssh succeeded")
+			log.Logf(2, "isolated: trying to reboot by ssh")
```

Comment:
I think these 2 log lines can be combined into 1 without losing any informativeness. Consider you have 10 devices and each of them prints 5 lines per reboot.
```
log.Logf(2, "isolated: ssh succeeded, trying to reboot by ssh")
```
looks as informative, but less lines in log and in source code.
If you want to improve logging, what would really be useful here is instance index, because currently it's total intermixed mess without possibility to understand what happens with which device.